_html/templates/form-work.php

```diff
@@ -0,0 +1,204 @@
+                <div class="field--type-string field--name-field-name field--widget-string-textfield js-form-wrapper form-wrapper" data-drupal-selector="edit-field-name-wrapper" id="edit-field-name-wrapper">      <div class="js-form-item form-item js-form-type-textfield form-type-textfield js-form-item-field-name-0-value form-item-field-name-0-value">
+                    <label for="edit-field-name-0-value" class="js-form-required form-required">Session/workshop title</label>
+                    <input class="js-text-full text-full form-text required" data-drupal-selector="edit-field-name-0-value" type="text" id="edit-field-name-0-value" name="field_name[0][value]" value="" size="60" maxlength="255" placeholder="" required="required" aria-required="true">
+                </div>
+
+        </div>
+        <input autocomplete="off" data-drupal-selector="form-7k1hz6i9v2hfppazllqvquaedkgwhwcuj8vyenm9c-u" type="hidden" name="form_build_id" value="form-7K1hZ6i9v2hFppAzLlqvquAedKGwhwCuj8VyENM9c-U">
+        <input data-drupal-selector="edit-contact-message-session-proposal-form-form-form-token" type="hidden" name="form_token" value="qeBEIqfh7IyuoCUBEkzCxzUnC_H_E-EsGxC9hEjck10">
+        <input data-drupal-selector="edit-contact-message-session-proposal-form-form" type="hidden" name="form_id" value="contact_message_session_proposal_form_form">
+        <div class="field--type-entity-reference field--name-field-user field--widget-entity-reference-autocomplete js-form-wrapper form-wrapper" data-drupal-selector="edit-field-user-wrapper" id="edit-field-user-wrapper"><div id="field-user-add-more-wrapper">  
+            <div class="js-form-item form-item">
+            <div class="tabledrag-toggle-weight-wrapper"><button type="button" class="link tabledrag-toggle-weight" title="Re-order rows by numerical weight instead of dragging.">Show row weights</button></div>
+            <div class="tableresponsive-toggle-columns"><button type="button" class="link tableresponsive-toggle" title="Show table cells that were hidden to make the table fit within a small screen." style="display: none;">Hide lower priority columns</button></div>
+            <table id="field-user-values" class="field-multiple-table responsive-enabled" aria-describedby="edit-field-user--description" data-striping="1">
+                <thead>
+                    <tr>
+                        <th colspan="2" class="field-label"><h4 class="label js-form-required form-required">Speakers</h4></th>
+                        <th class="tabledrag-hide" style="display: none;">Order</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr class="draggable odd">
+                        <td class="field-multiple-drag"><a href="#" class="tabledrag-handle" title="Drag to re-order"><div class="handle">&nbsp;</div></a></td>
```

Comment:
Please check in all browser to see if the div elements are displayed properly inside td.